config/routes.rb

```diff
@@ -2,16 +2,17 @@
   root "locations#index"
 
   resources :profiles, only: [:index, :new, :create, :show, :edit, :update]
-  resources :locations
+  # resources :locations
```

Comment:
Don't leave comments in your code like this. Either you use something or you don't :D